### PR TITLE
Update installation.md

### PR DESCRIPTION
**Pull request by Verron**
Add additional details to use the included config and schema in Lumen.

- [ ] Added or updated tests
- [ ] Documented user facing changes
- [ ] Updated CHANGELOG.md

**Changes**

**Breaking changes**

### CODE REVIEW

File: docs/5/getting-started/installation.md [hunk 1]
```diff
@@ -19,10 +19,31 @@ php artisan vendor:publish --tag=lighthouse-schema
 
 ## Lumen
 
-Register the service provider in your `bootstrap/app.php` file:
+To use the included lighthouse config, copy it over to your config folder.
+
+```bash
+mkdir -p config
+cp vendor/nuwave/lighthouse/src/lighthouse config/
```
Comment (spawnia) [inline docs/5/getting-started/installation.md:26]:
```suggestion
cp vendor/nuwave/lighthouse/src/lighthouse.php config/
```

File: docs/5/getting-started/installation.md [hunk 2]
```diff
@@ -19,10 +19,31 @@ php artisan vendor:publish --tag=lighthouse-schema
...
+```
+
+To get you going right away in Lumen, copy over the included default schema.
+
+```bash
+mkdir -p graphql
+cp vendor/nuwave/lighthouse/src/default-schema.graphql graphql/schema.graphql
+```
+
+Register these service providers in your `bootstrap/app.php` file:
```
Comment (spawnia) [inline docs/5/getting-started/installation.md:42]:
```suggestion
Register the service provider in your `bootstrap/app.php` file:
```

File: docs/5/getting-started/installation.md [hunk 3]
```diff
@@ -19,10 +19,31 @@ php artisan vendor:publish --tag=lighthouse-schema
...
 
 ```php
 $app->register(\Nuwave\Lighthouse\LighthouseServiceProvider::class);
+$app->register(\Nuwave\Lighthouse\Pagination\PaginationServiceProvider::class);
```
Comment (spawnia) [inline docs/5/getting-started/installation.md:46]:
```suggestion
```

The section below describes how to add supplementary service providers

**Review by spawnia — CHANGES_REQUESTED** [review]
Great job overall

File: docs/5/getting-started/installation.md (hunk 3 above)
Comment (Verron) [reply docs/5/getting-started/installation.md:46]: @spawnia Yep, I agree on the sections below specifying supplementary service provides. With the addition of recommending a copying of default-schema.graphql and lighthouse.php (setup parity between Lumen and Full Framework), following the documentation with a fresh install will result in an error on the @paginate directive.

```
No directive found for `paginate`
```

I wouldn't be surprised if others overlook the section below as well :(

Comment (spawnia) [reply docs/5/getting-started/installation.md:46]: This issue prompted me to reconsider the default schema. At first, I wanted to go with a slimmed down version that does not use extra service providers, but found it more useful to have a fully-featured schema that utilizes appropriate features. Feel free to review https://github.com/nuwave/lighthouse/pull/2032.

I would propose to keep this section with the single service provider. Instead, let's move the part about publishing the default schema after it, and then mention the extra required service providers.

Comment (spawnia) [reply docs/5/getting-started/installation.md:46]: Improved with https://github.com/nuwave/lighthouse/commit/6f261ed71112f1ae64fb3088cfeae3a9d4518ad0

**Pull request merged**